File: Modificador de 4 lenguaje/js/clasenormales/Libro.js
class Libro {
    constructor(titulo, autor) {
        this.titulo = titulo;
        this.autor = autor;
    }

    static copiar(libroExistente) {
        return new Libro(libroExistente.titulo, libroExistente.autor);
    }

    mostrar() {
        console.log(`Libro: "${this.titulo}" de ${this.autor}`);
    }
}

module.exports = Libro;

File: Modificador de 4 lenguaje/C#/ClaseNormales/ClasesNormales/Libro.cs
namespace EjemploConstructores.ClasesNormales
{
    class Libro
    {
        public string Titulo;

        // Constructor con parámetros (único)
        public Libro(string titulo)
        {
            Titulo = titulo;
        }

        public void Mostrar()
        {
            System.Console.WriteLine($"Libro: {Titulo}");
        }
    }
}
